sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -246,6 +250,8 @@ void WinHttpTransport::CreateRequestHandle(std::unique_ptr<_detail::HandleManage
   const std::string& path = handleManager->m_request.GetUrl().GetRelativeUrl();
   HttpMethod requestMethod = handleManager->m_request.GetMethod();
 
+  handleManager->m_context.ThrowIfCancelled();
```

Comment:
This one I think we don't need it as well:

https://docs.microsoft.com/en-us/windows/win32/api/winhttp/nf-winhttp-winhttpopenrequest

The WinHttpOpenRequest function creates an HTTP request handle.